charts/stable/plex/values.yaml

```diff
@@ -19,7 +19,7 @@ persistence:
 plex:
   serverURL: "{{ .Values.service.main.ports.main.protocol | lower }}://{{ .Values.plex.serverIP }}:{{ .Values.service.main.ports.main.port }}"
   serverIngressURL: "{{ if .Values.ingress.main.enabled }}{{ with (first .Values.ingress.main.hosts) }},https://{{ .host }}{{ end }}{{ end }}"
-  serverIngressURLPort: "{{ if .Values.ingress.main.enabled }}{{ with (first .Values.ingress.main.hosts) }},https://{{ .host }}:{{ .Values.service.main.ports.main.port }}{{ end }}{{ end }}"
+  serverIngressURLPort: "{{ if .Values.ingress.main.enabled }}{{ $port := .Values.service.main.ports.main.port }}{{ with (first .Values.ingress.main.hosts) }},https://{{ .host }}:{{ $port }}{{ end }}{{ end }}"
```

Comment:
I think this is not necessary. The port is on the host which is not for ingress through the traefik reverse proxy. Keeping the first two URLs is enough.

Reply:
The url should resolve to your public IP, on which you can have the port (eg. 32400) forwarded. 
This covers any case.

It's still evaluated after 443. If 443 works plex will just ignore the next

Reply:
If I set up port forwarding of the internal service port, I don’t need to set up ingress to expose my service on public network. The simple network service already exposes my service on the port in my LAN IP. The benefit of having ingress is that it allows me to set up only one port forwarding (443 for https) and support multiple services using different hosts on the reverse proxy.